src/organization-domains/organization-domains.spec.ts

```diff
@@ -0,0 +1,100 @@
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import { WorkOS } from '../workos';
+import getOrganizationDomainPending from './fixtures/get-organization-domain-pending.json';
+import getOrganizationDomainUnverified from './fixtures/get-organization-domain-unverified.json';
+import { VerificationState } from './interfaces';
+
+const mock = new MockAdapter(axios);
+const workos = new WorkOS('sk_test_Sz3IQjepeSWaI4cMS4ms4sMuU');
+
+describe('OrganizationDomains', () => {
+  afterEach(() => mock.resetHistory());
+
+  describe('get', () => {
+    it(`requests an Organization Domain`, async () => {
```

Comment:
Just a small nit, we don't need string templates if we're not interpolating anything.
```suggestion
    it('requests an Organization Domain', async () => {
```